Tests/FunctionalTests/Sources/FTRSwiftTests.swift

```diff
@@ -29,15 +29,15 @@ class TextFieldEventsRecorder {
   func registerActionBlock() -> GREYActionBlock {
     NotificationCenter.default.addObserver(self,
                                            selector: #selector(textDidBeginEditingHandler),
-                                           name: NSNotification.Name.UITextFieldTextDidBeginEditing,
+                                           name: UITextField.textDidBeginEditingNotification,
```

Comment:
Can we abstract this behind an if swift(>=5.0)  check since the Travis tests are still using the older version?

Reply:
Done.